packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add optional approval request when calling `addTransactionBatch` ([#5793](https://github.com/MetaMask/core/pull/5793))
+  - Add `transactionBatches` array to state.
+  - Add `TransactionBatchMeta` type.
+  - Add `addBatchMetadata` to store batch metadata and `wipeTransactionBatches` to clean up state after batch hook completion.
```

Comment:
The changelog is only for public facing changes, so things exported in `index.ts`.